requirements.txt

```diff
@@ -1 +1,2 @@
-bs4~=0.0.1
\ No newline at end of file
+bs4~=0.0.1
+appscript~=1.0.1
```

Comment:
hey so this can't go into the requirements for everybody right? since its just needed for mac. make a `mac-requirements.txt` and mention that you have to run `pip install -r mac-requirements.txt` in the README.md set up section :)